config/tech-docs.yml

```diff
@@ -3,7 +3,7 @@ host: https://tech-docs.hmpps.service.justice.gov.uk
 
 # Header-related options
 show_govuk_logo: false
-service_name: HMPPS Tech Team docs
+service_name: DPS Tech Team docs
```

Comment:
why DPS?

Reply:
I heard that the HMPPS Tech Team is not to be.... yet.